ui/src/components/Dashboard/FilterBar.js

```diff
@@ -150,7 +150,7 @@ const FilterBar = ({
       }
     });
 
-    updateFilters(filters);
+    setFilters(filters);
```

Comment:
update filters updates the history as well incase of bad filter passed at the url.

Reply:
what is a bad filter? this code removes the table filters. 
If we have a "bad" filter the code will not use them